sqlmesh/core/plan/evaluator.py

```diff
@@ -136,7 +136,7 @@ def _push(self, plan: Plan) -> None:
         snapshot_id_to_snapshot = {s.snapshot_id: s for s in plan.snapshots}
 
         if plan.new_snapshots:
-            self.console.start_creation_progress(len(plan.new_snapshots))
+            self.console.start_creation_progress(len([s for s in plan.new_snapshots if s.is_model]))
```

Comment:
What if the plan only had new snapshots that were not models? Would we print 0 here?

Reply:
Good call, I'll need to check that but it might be the case yes

Reply:
> What if the plan only had new snapshots that were not models?

@eakmanrq how can I reach this state to test the behavior? E.g. run plan and then add an audit and re-run plan?

Reply:
> What if the plan only had new snapshots that were not models? Would we print 0 here?

@eakmanrq I assume the intended behavior then would be to not display a progress bar at all, correct? That is, we don't version non-model snapshots. If that's what you meant then I believe it should be addressed with the latest change I made.